lib/plugins/aws/index.js

```diff
@@ -59,9 +59,19 @@ class SDK {
       // req.on('send', function (r) {console.log(r)});
 
       return new BbPromise((resolve, reject) => {
-        req.send((err, data) => {
+        req.send((errParam, data) => {
+          const err = errParam;
           if (err) {
-            reject(err);
+            if (err.message === 'Missing credentials in config') {
+              const errorMessage = [
+                '"default" profile was not found in ~/.aws/credentials.',
+                ' Please make sure you add a "default" profile',
+                ' with valid API Key & Secret, or set the API Key & Secret',
+                ' as env vars. For more info, check our docs.',
```

Comment:
'For more info, check our docs' sounds a bit too direct imho. 'You can get more details in our docs' sounds a bit friendlier imho